src/3d/Viewport3d.tsx

```diff
@@ -138,7 +142,7 @@ const Viewport3d = () => {
             <Leva hidden={window.location.hash !== '#debug'} />
             <Canvas shadows={true} onKeyDown={handleKeyPress}>
                 <color attach="background" args={[0.9, 0.9, 0.9]} />
-                <OrbitControls target={cameraTarget} />
+                <OrbitControls target={cameraTarget} onStart={() => { dispatch(toggleOrbiting()) }} onEnd={() => { dispatch(toggleOrbiting()) }} />
```

Comment:
ahhhh thanks for solving this one. 
Do you mind If I extract your fix and make it a separate PR?

Reply:
sure, no problem

Reply:
theres a bit of a performance hit doing this via a state. Might need to investage a different solution. I wonder if the faces can store a pointer to the orbit controls and listen directly?

https://user-images.githubusercontent.com/25563790/201840946-89ad4698-4c7d-46bc-965c-e69f9230a4f0.mp4

Reply:
I gave it a go using references, but I might have a simpler option... track the delta of the mouse on mouseDown

Reply:
Back to references... PR here https://github.com/AlasdairMott/block-builder/pull/7